.cookiecutter/includes/README/head.md

```diff
@@ -0,0 +1,38 @@
+The `gh pr-upsert` command will create a new pull request (PR) with the commits
+on your current branch, generating a PR title and body from your commit
+messages:
+
+```terminal
+$ gh pr-upsert
+There's no PR for this branch yet, creating one
+```
+
+Unlike `gh pr create`, `gh pr-upsert` always pushes your branch directly to the
+GitHub repo before opening a PR. It's a non-interactive command and doesn't ask
+you whether or not to push or offer to create a fork.
+
+Also unlike `gh pr create`, if there's already a PR for the branch then `gh
+pr-upsert` will force push it:
+
+```terminal
+$ gh pr-upsert
+There's already an open PR for this branch
+Your local changes are different from the open PR
+No one else has pushed to the PR, updating it
+```
+
+Requires [Git](https://git-scm.com/) and [GitHub CLI](https://cli.github.com/)
+to be installed.
+
+To install the latest version of `gh-pr-upsert` as a GitHub CLI extension run:
+
+```terminal
+gh extension install hypothesis/gh-pr-upsert
+```
+
+Then to upgrade to a new version run `gh extension upgrade gh-pr-upsert`.
+To remove the extension run `gh extension remove gh-pr-upsert`.
+
+Alternatively you can install `gh-pr-upsert` as a Python package and the
+command will be `gh-pr-upsert` rather than `gh pr-upsert`. See below for
+details.
```

Comment:
This is a little awkward. If you look at [the complete README](https://github.com/hypothesis/gh-pr-upsert/blob/initial-implementation/README.md) you'll see that this head section at the top explains how to install gh-pr-upsert as a GitHub CLI extension and then it's immediately followed by the title **Installing** from the cookiecutter and the instructions for installing it as a Python package. Both methods actually work fine but just the flow of the README is weird. May want to add an `installing.md` include to the cookiecutter so that this repo can override the standard **Installing** section and provide its own section that explains that there are two ways of installing it.